server/build.gradle

```diff
@@ -74,57 +74,43 @@ dependencies {
   compileOnly project(':libs:opensearch-plugin-classloader')
   testRuntimeOnly project(':libs:opensearch-plugin-classloader')
 
-  // lucene
-  api "org.apache.lucene:lucene-core:${versions.lucene}"
-  api "org.apache.lucene:lucene-analysis-common:${versions.lucene}"
-  api "org.apache.lucene:lucene-backward-codecs:${versions.lucene}"
-  api "org.apache.lucene:lucene-grouping:${versions.lucene}"
-  api "org.apache.lucene:lucene-highlighter:${versions.lucene}"
-  api "org.apache.lucene:lucene-join:${versions.lucene}"
-  api "org.apache.lucene:lucene-memory:${versions.lucene}"
-  api "org.apache.lucene:lucene-misc:${versions.lucene}"
-  api "org.apache.lucene:lucene-queries:${versions.lucene}"
-  api "org.apache.lucene:lucene-queryparser:${versions.lucene}"
-  api "org.apache.lucene:lucene-sandbox:${versions.lucene}"
-  api "org.apache.lucene:lucene-spatial-extras:${versions.lucene}"
-  api "org.apache.lucene:lucene-spatial3d:${versions.lucene}"
-  api "org.apache.lucene:lucene-suggest:${versions.lucene}"
+  api libs.bundles.lucene
 
   // utilities
   api project(":libs:opensearch-cli")
 
   // time handling, remove with java 8 time
-  api "joda-time:joda-time:${versions.joda}"
+  api libs.jodatime
```

Comment:
I :100:  agree that replacing all in one shot is certainly too much, but I would suggest to pick one (or a few, if it makes sense) dependency at a time and replace it everywhere, not only in `server` module. Does it make sense to you?

Reply:
I'd also be comfortable doing this library-by-library as well. i.e. A PR to upgrade log4j, another for lucene, etc. Let me know what you think.

Reply:
> I'd also be comfortable doing this library-by-library as well. i.e. A PR to upgrade log4j, another for lucene, etc. Let me know what you think.

I think both approaches are fine, definitely don't want you to do more work (thanks for pushing it)., Let's finish up the `server` module than, I still see some deps hardcoded ('org.roaringbitmap:RoaringBitmap:1.3.0') which we probably should move to catalog as well?

Reply:
> Should we define all dependencies in the version catalog or just dependencies that we want to keep sync'ed across modules with the same version of the same library?

I think our goal is to publish the version catalog at some point, at least for share modules (like server fe), it would be good to have versions in the catalog (and some plugins could pick them up later on).

Reply:
Adding roaring bitmap to the version catalog.